### PR TITLE
use os.path.join instead of setting path {},{}

### PR DESCRIPTION
**Pull request by sometheycallme**

### CODE REVIEW

**Comment by kropped** [comment]
:bell: Staged `cleanerbot_test_gitrepo` at revision cf7d2eea18c6f59465f6d95e6a72ffa7c74fa305 on 7947bbdd3abc

**Comment by sometheycallme** [comment]
@jumanjiman 

Testing pipeline we'll let autostager stage this and I'll try to see if the webserver works.

**Comment by sometheycallme** [comment]
much better:

```
core@pipeline ~ $ docker ps
CONTAINER ID        IMAGE                     COMMAND                  CREATED             STATUS              PORTS                     NAMES
7947bbdd3abc        autostager                "python autostager/au"   2 minutes ago       Up 2 minutes                                  adoring_dijkstra
0d6b34f8fc55        ansible-controller        "python webserver/ser"   2 minutes ago       Up 2 minutes        0.0.0.0:8080->8080/tcp    small_jang
```

Webserver on ansible-controller is up now alongside autostager

now to continue to build a test with #81

File: webserver/server.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def do_POST(self):
 		    return
 	    directory = post_data['git_handle'] + '/' + post_data['branch_name']
 	    safe_dir = self.alphafy(directory)
-	    path = "{0}/{1}".format(base_dir, safe_dir)
+	    path = os.path.join(base_dir, safe_dir)
 	    # Use this to set ANSIBLE_HOSTS environment variable as base_dir, safe_dir
 	    # and change into path
 	    os.putenv('ANSIBLE_HOSTS',"{0}/{1}/hosts".format(base_dir,safe_dir))
```
Comment (jumanjiman) [inline webserver/server.py]: use the path join here, too! 😁

**Comment by sometheycallme** [comment]
>use the path join here, too!

seems like another good spot.

**Comment by sometheycallme** [comment]
Pipeline and devenv coming down.  going out for lunch :)

**Pull request merged**